Core/src/org/sleuthkit/autopsy/modules/encryptiondetection/EncryptionDetectionFileIngestModule.java

```diff
@@ -155,6 +160,9 @@ public void startUp(IngestJobContext context) throws IngestModule.IngestModuleEx
                  */
                 String mimeType = fileTypeDetector.getMIMEType(file);
                 if (mimeType.equals("application/octet-stream") && isFileEncryptionSuspected(file)) {
+                    if (checkFileLocationExtension(file)) {
+                        return flagFile(file, BlackboardArtifact.ARTIFACT_TYPE.TSK_ENCRYPTION_DETECTED, Bundle.EncryptionDetectionFileIngestModule_artifactComment_location());
+                    }
                     return flagFile(file, BlackboardArtifact.ARTIFACT_TYPE.TSK_ENCRYPTION_SUSPECTED,
```

Comment:
I believe flagging them as encryption suspected should be in an else condition so we are not flagging them as both Encryption Detected and Encryption Suspected.

Reply:
Since it is doing a return in the if (163/164) will it ever get to the second return (166)?

Reply:
good point, really wondering why I have what looks like both artifacts for them then.

Reply:
Are you using the dataset I provided?  If so look closer at the files I changed name slightly and put in a different directory.

Reply:
Oh they are different files, my bad. yes working as intended.